examples/gno.land/r/sys/params/params.gno

```diff
@@ -1,54 +0,0 @@
-// Package params provides functions for creating parameter executors that
```

Comment:
Ditto, I don't think these should be deleted.

Reply:
reverted it